src/dotnet/commands/dotnet-pack/LocalizableStrings.cs

```diff
@@ -2,29 +2,29 @@
 {
     internal class LocalizableStrings
     {
-        public const string AppFullName = "pack";
+        public const string AppFullName = ".NET Core NuGet Package Packer";
 
         public const string AppDescription = "pack for msbuild";
 
         public const string CmdOutputDir  = "OUTPUT_DIR";
 
-        public const string CmdOutputDirDescription = "Directory in which to place outputs";
+        public const string CmdOutputDirDescription = "Directory in which to place built packages.";
 
-        public const string CmdNoBuildOptionDescription = "Do not build project before packing";
+        public const string CmdNoBuildOptionDescription = "Skip building the project prior to packing. By default, the project will be built.";
 
-        public const string CmdIncludeSymbolsDescription = "Include PDBs along with the DLLs in the output folder";
+        public const string CmdIncludeSymbolsDescription = "Include packages with symbols in addition to assembly (DLL) packages in output directory.";
```

Comment:
Create Symbols Packages in addition to regular Packages. By default, only regular Packages are created.

?

I checked nuget docs. They don't seem to have a better name for regular packages.